packages/electron-builder-squirrel-windows/src/squirrelPack.ts

```diff
@@ -1,295 +0,0 @@
-      if (line.includes(packageName)) {
-        return line.trim()
-      }
-    }
-
-    throw new Error(`Invalid output, cannot find last release entry, output: ${out}`)
-  }
-
-  private async createEmbeddedArchiveFile(nupkgPath: string, dirToArchive: string) {
-    const embeddedArchiveFile = await this.packager.getTempFile("setup.zip")
-    const path7za = await getPath7za()
-    await exec(
-      path7za,
-      compute7zCompressArgs("zip", {
-        isRegularFile: true,
-        compression: this.packager.compression,
-      }).concat(embeddedArchiveFile, "."),
-      {
-        cwd: dirToArchive,
-      }
-    )
-    await exec(
-      path7za,
-      compute7zCompressArgs("zip", {
-        isRegularFile: true,
-        compression: "store" /* nupkg is already compressed */,
-      }).concat(embeddedArchiveFile, nupkgPath)
-    )
-    return embeddedArchiveFile
-  }
-}
-
-async function pack(options: SquirrelOptions, directory: string, updateFile: string, outFile: string, version: string, packager: WinPackager) {
-  // SW now doesn't support 0-level nupkg compressed files. It means that we are forced to use level 1 if store level requested.
-  const archive = archiver("zip", { zlib: { level: Math.max(1, options.packageCompressionLevel == null ? 9 : options.packageCompressionLevel) } })
-  const archiveOut = createWriteStream(outFile)
-  const archivePromise = new Promise((resolve, reject) => {
-    archive.on("error", reject)
-    archiveOut.on("error", reject)
-    archiveOut.on("close", resolve)
-  })
-  archive.pipe(archiveOut)
-
-  const author = options.authors
-  const copyright = options.copyright || `Copyright © ${new Date().getFullYear()} ${author}`
-  const nuspecContent = `<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
-  <metadata>
-    <id>${options.appId}</id>
-    <version>${version}</version>
-    <title>${options.productName}</title>
-    <authors>${author}</authors>
-    <iconUrl>${options.iconUrl}</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>${options.description}</description>
-    <copyright>${copyright}</copyright>${options.extraMetadataSpecs || ""}
-  </metadata>
-</package>`
-  debug(`Created NuSpec file:\n${nuspecContent}`)
-  archive.append(nuspecContent.replace(/\n/, "\r\n"), { name: `${options.name}.nuspec` })
-
-  //noinspection SpellCheckingInspection
-  archive.append(
-    `<?xml version="1.0" encoding="utf-8"?>
-<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
-  <Relationship Type="http://schemas.microsoft.com/packaging/2010/07/manifest" Target="/${options.name}.nuspec" Id="Re0" />
-  <Relationship Type="http://schemas.openxmlformats.org/package/2006/relationships/metadata/core-properties" Target="/package/services/metadata/core-properties/1.psmdcp" Id="Re1" />
-</Relationships>`.replace(/\n/, "\r\n"),
-    { name: ".rels", prefix: "_rels" }
```

Comment:
Where did this go? I don't see it in our template or in `electron-winstaller`
https://github.com/electron/windows-installer/blob/e722a519f8625c939087de2602b2b5b594886bc4/vendor/template.wxs

Reply:
The windows-installer calls the nuget command to generate the nupkg file, and the nuget command will automatically generate the .rels. The specific code is sourced from

https://github.com/NuGet/NuGet.Client/blob/fa6bd3682bb698604a09d63601a708f2b357b7a8/src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs#L1274